english/light-fire/graham-crackers/chocolate/find-chocolate.md

```diff
@@ -1,11 +1,2 @@
 You don't find any chocolate in your jacket, but you do find some pre-fermented and dried cocoa beans and some sugar.
 
-You realize that you could make your own chocolate out of these ingredients,
-but it would probably take a while. You take a moment to weigh your options.
-
-You decide to:
-
-[Be lazy](../eat-goo/eat-goo.md) (just eat the graham crackers and marshmallow goo already)
-
-[Be patient](make-chocolate/make-chocolate.md) (try to make chocolate)
-
```

Comment:
You're welcome! Good luck getitng it sorted out. I know these types of situations can be very confusing.